conmon-rs/server/src/container_log.rs

```diff
@@ -13,36 +13,43 @@ pub struct ContainerLog {
     drivers: Vec<LogDriver>,
 }
 
-#[derive(Debug)]
```

Comment:
Why does deriving `Debug` not work? It should :thinking:

Reply:
It didn't worked as the code changed cuz of merge conflicts, needs to look and use it as it did before...